examples/imitation_learning_openbot/generate_dataset.py

```diff
@@ -11,12 +11,18 @@
 import os
 import pandas as pd
 import shutil
-import spear
```

Comment:
I like the `spear` import being up here. It is a "lower-level" import than these other example-specific imports.

Reply:
Updated this.